integration-testing/examples/mpp-version-catalog/shared/build.gradle.kts

```diff
@@ -8,6 +12,7 @@ buildscript {
 
 repositories {
     mavenCentral()
+    maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/dev")
```

Comment:
Could you please explain why do we need `dev` repo here and in other examples? I hight don't recommend to use this repo, because it has temporary artifacts, which don't live long.
So if it not expected by TeamCity build to have, I would suggest to remove the usages of this repo.

Reply:
I guess, there were some cases, when I had to run `kotlinx-atomicfu` build against some Kotlin version published to `space.dev`, that's why I left them. Is that a problem or it's better to remove this repo anyway?

Reply:
There are some points to reduce the usage of this repository:
- decrease the transfer of unnecessary traffic via the network
- decrease failures connected with unstable internet connection on agents
I recommend using the `maven.zip` artifact in our CI builds instead of this. Of course, we can create an additional issue for that.